test/test_prototype_transforms_consistency.py

```diff
@@ -1,16 +1,26 @@
 import enum
 import inspect
+from importlib.machinery import SourceFileLoader
+from pathlib import Path
 
 import numpy as np
 import PIL.Image
 import pytest
 
 import torch
-from prototype_common_utils import ArgsKwargs, assert_equal, make_images
+from prototype_common_utils import (
+    ArgsKwargs,
+    assert_equal,
+    make_bounding_box,
+    make_detection_mask,
+    make_image,
+    make_images,
+    make_label,
+)
 from torchvision import transforms as legacy_transforms
 from torchvision._utils import sequence_to_str
 from torchvision.prototype import features, transforms as prototype_transforms
-from torchvision.prototype.transforms.functional import to_image_pil
+from torchvision.prototype.transforms.functional import to_image_pil, to_pil_image
```

Comment:
Why do we need both? Isn't the one just aliased to the other? I think `to_image_pil` is the new canonical name

Reply:
```suggestion
from torchvision.prototype.transforms.functional import to_image_pil
```